CONTRIBUTING.md

```diff
@@ -285,11 +285,11 @@ The reviewer may ask you to later squash the commits together to provide
 a clean commit history before merging in the feature.
 
 It's important to write a proper commit title and description. The commit title
-should fir around 50 characters; it is the first line of the commit text. The
+should fire around 50 characters; it is the first line of the commit text. The
 second line of the commit text must be left blank. The third line and beyond is
 the commit message. You should write a commit message. If you do, wrap all
 lines at 72 characters. You should explain what the commit does, what
-references you used, and any other information that helps understanding your
+references you used, and any other information that helps to understanding your
```

Comment:
it's either 'information that helps understanding' or 'information that helps to understand', but not 'to understanding' I think.